components/compliance-service/scripts/create-pg-data.sh

```diff
@@ -88,7 +107,7 @@ for i in $(seq 1 ${job_count}); do
       "name": "my job",
       "tags": [],
       "type": "exec",
-      "nodes": ['${vagrant_node_id}', '${ec2_node_id}'],
+      "nodes": ['${vagrant_node_id}', '${ec2_node_id}', '${docker_node_id}'],
```

Comment:
hmm. i might remove the docker node id from the scan job given that i assume it will always be unreachable